resources/serverless/templates/function-usage-kind.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.kyma-project.io/v1alpha1
+kind: UsageKind
+metadata:
+  name: function
```

Comment:
I would rather call it `serverless-function` to avoid the issue that we had for 1.12 - deletion of `function` was blocked because it was still used.

Reply:
Changed.